test/spec/MessagesSpec.js

```
@@ -63,6 +62,15 @@ describe('Messages should work', function () {
     });
   };
 
+  // Assure that the list of messages avaiable from the API is empty
+  const clear_messages = () => {
```

ESLint

[error] Identifier 'clear_messages' is not in camel case.

(camelcase)